src/utils/setup/serialize-search-rss-data.js

```diff
@@ -0,0 +1,25 @@
+const getCustomRSSElements = article => {
```

Comment:
Here we want to add some "custom" elements to the feed. We also just want to exclude a tag outright if it has 0 entries (such as no tags for whatever reason). If we don't intentionally exclude, the XML gives an odd effect of just giving `</tagname>` which causes parsing issues.